sdk/ml/azure-ai-ml/tests/environment/unittests/test_environment_operations_registry.py

```diff
@@ -0,0 +1,60 @@
+from unittest.mock import Mock, patch
```

Comment:
`patch` import is not used

Reply:
since these are tests, I will take care of this in the next PR